pkg/api/azenvtypes.go

```diff
@@ -36,10 +36,12 @@ type DCOSSpecConfig struct {
 
 //KubernetesSpecConfig is the kubernetes container images used.
 type KubernetesSpecConfig struct {
-	AzureTelemetryPID                string `json:"azureTelemetryPID,omitempty"`
-	KubernetesImageBase              string `json:"kubernetesImageBase,omitempty"`
-	TillerImageBase                  string `json:"tillerImageBase,omitempty"`
-	ACIConnectorImageBase            string `json:"aciConnectorImageBase,omitempty"`
+	AzureTelemetryPID string `json:"azureTelemetryPID,omitempty"`
+	// KubernetesImageBase defines a base image URL substring to source images that originate from upstream k8s.gcr.io
```

Comment:
All of this amounts to my making some explanatory comments about the differences between `KubernetesImageBase` and `MCRKubernetesImageBase`